packages/commonwealth/client/scripts/views/pages/CommunityManagement/Contests/ManageContest/steps/SignTransactionsStep/SignTransactionsStep.tsx

```diff
@@ -36,6 +36,7 @@ interface SignTransactionsStepProps {
 
 const SEVEN_DAYS_IN_SECONDS = 60 * 60 * 24 * 7;
 const FIVE_MINS_IN_SECONDS = 60 * 5;
+const ZERO_ADDRESS = '0x0000000000000000000000000000000000000000';
```

Comment:
can we move this to `lib/shared`, i see this address is used in a lot of other code areas

Reply:
done